cli/coda.ts

```diff
@@ -83,6 +84,18 @@ if (require.main === module) {
       },
       handler: handleRegister,
     })
+    .command({
+      command: 'whoami [apiToken]',
+      describe: 'Looks up the api token owner',
```

Comment:
We're pretty disciplined about language/style/punctuation in developer-facing documentation. Would probably do `Looks up information about the API token that is registered in this environment.`

Reply:
Changed the wording but didn't include the trailing period for consistency with all the other one-sentence command descriptions